printer.go

```diff
@@ -54,13 +57,46 @@ type DRIVER_INFO_8 struct {
 	MinInboxDriverVerVersion uint32
 }
 
+type JOB_INFO_1 struct {
+	JobID        uint32
+	PrinterName  *uint16
+	MachineName  *uint16
+	UserName     *uint16
+	Document     *uint16
+	DataType     *uint16
+	Status       *uint16
+	StatusCode   uint32
+	Priority     uint32
+	Position     uint32
+	TotalPages   uint32
+	PagesPrinted uint32
+	Submitted    syscall.Systemtime
+}
+
 const (
 	PRINTER_ENUM_LOCAL       = 2
 	PRINTER_ENUM_CONNECTIONS = 4
 
 	PRINTER_DRIVER_XPS = 0x00000002
 )
 
+const (
+	JOB_STATUS_PAUSED            = 0x00000001 // Job is paused
+	JOB_STATUS_ERROR             = 0x00000002 // An error is associated with the job
+	JOB_STATUS_DELETING          = 0x00000004 // Job is being deleted
+	JOB_STATUS_SPOOLING          = 0x00000008 // Job is spooling
+	JOB_STATUS_PRINTING          = 0x00000010 // Job is printing
+	JOB_STATUS_OFFLINE           = 0x00000020 // Printer is offline
+	JOB_STATUS_PAPEROUT          = 0x00000040 // Printer is out of paper
+	JOB_STATUS_PRINTED           = 0x00000080 // Job has printed
+	JOB_STATUS_DELETED           = 0x00000100 // Job has been deleted
+	JOB_STATUS_BLOCKED_DEVQ      = 0x00000200 // Printer driver cannot print the job
+	JOB_STATUS_USER_INTERVENTION = 0x00000400 // User action required
+	JOB_STATUS_RESTART           = 0x00000800 // Job has been restarted
+	JOB_STATUS_COMPLETE          = 0x00001000 // Job has been delivered to the printer
+	JOB_STATUS_RETAINED          = 0x00002000 // Job has been retained in the print queue
```

Comment:
JOB_STATUS_RENDERING_LOCALLY ?

from winspool.h
```
#if (NTDDI_VERSION >= NTDDI_VISTA)
    #define JOB_STATUS_RETAINED             0x00002000
    #define JOB_STATUS_RENDERING_LOCALLY    0x00004000
#endif // (NTDDI_VERSION >= NTDDI_VISTA)
```

Reply:
Thanks. I'll add it. That wasn't listed on https://docs.microsoft.com/en-us/windows/desktop/printdocs/job-info-1